compiler-core/src/language_server/tests/action.rs

```diff
@@ -52,6 +52,7 @@ fn engine_response(src: &str, line: u32) -> engine::Response<Option<Vec<lsp_type
 
 const REMOVE_UNUSED_IMPORTS: &str = "Remove unused imports";
 const REMOVE_REDUNDANT_TUPLES: &str = "Remove redundant tuples";
+const DESUAR_USE_EXPRESSION: &str = "Desugar use expression";
```

Comment:
Seems to be a typo (will need fixing elsewhere too):

```suggestion
const DESUGAR_USE_EXPRESSION: &str = "Desugar use expression";
```

Reply:
done!